confluent_kafka/avro/__init__.py

```diff
@@ -1,6 +1,8 @@
 import logging
 import sys
 
+from confluent_kafka.cimpl import Producer
```

Comment:
Are you sure it should import from .cimpl directly and not from confluent.kafka?